README.md

```diff
@@ -10,15 +10,15 @@ Install the required libraries given in [requirements.txt](requirements.txt).
 
 Run the script:
 ```
-python download_plugins.py jenkins_plugins.yaml /path/to/download/directory
+python3 download_plugins.py jenkins_plugins.yaml /path/to/download/directory
 ```
 
 ### Docker Method
 Build an image and run the container:
 ```
-docker build -t jenkins-pluginator
+docker build -t jenkins-pluginator .
 docker run --rm \
-    -v jenkins_plugins.yaml:/usr/src/app/jenkins_plugins.yaml:ro \
+    -v "$PWD"/jenkins_plugins.yaml:/usr/src/app/jenkins_plugins.yaml:ro \
     -v /path/to/download/directory:/usr/src/app/jenkins_plugins \
     jenkins-pluginator
 ```
```

Comment:
It is kind of debatable, some people claim that when documenting terminal commands, we should avoid using `$` or `#` these prefixes so that the commands can be pasteable. However, others say that it has historical meanings and it is also helpful for the readability. I personally prefer having `$`, what do you think? cc. @jaredkosanovic @ianbrown9475  @adpextwindong

Reply:
For example

```
$ docker run foo
```
v.s.

```
docker run bar
```

Reply:
I agree that including `$` would be more helpful.